solidity/contracts/liquidity-protection/LiquidityProtectionSettings.sol

```diff
@@ -29,11 +28,17 @@ contract LiquidityProtectionSettings is ILiquidityProtectionSettings, AccessCont
     // list of whitelisted pools
     EnumerableSet.AddressSet private _poolWhitelist;
 
+    // list of subscribers
+    EnumerableSet.AddressSet private _subscriberList;
```

Comment:
`_subscribers` is enough, we don't usually use "lists" in our terminology

Reply:
Fixed.